src/Identity/UI/src/IdentityBuilderUIExtensions.cs

```diff
@@ -36,5 +63,96 @@ public static IdentityBuilder AddDefaultUI(this IdentityBuilder builder)
 
             return builder;
         }
+
+        private static Assembly GetApplicationAssembly(IdentityBuilder builder)
+        {
+            // This is the same logic that MVC follows to find the application assembly.
```

Comment:
@HaoK Can you link to the existing logic? Having trouble finding it by copy/pasting the source here.